Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -820,6 +818,14 @@ private bool CheckForCollisionBetween()
                     Map.debugDrawer.FlashCell(cell, 1, "o");
                 }
             }
+            if (possibleIntersections.Count > 0)
+            {
```

Comment:
This block needs to only run if `collided` is still `false`.  Otherwise we'll hit the pawns within this segment *and* hit the shield.

Reply:
It won't be true in current version til this blocked called

Reply:
Line 812 can still set it to true.

Reply:
Checked it again and found only one set to true